### PR TITLE
[TACHYON-1671] Rename tfs/mTFS/sTFS in shell and tests

### PR DESCRIPTION
**Pull request by calvinjia**
Final part of #2572

### CODE REVIEW

**Comment by jsimsa** [comment]
LGTM

**Comment by aaudiber** [comment]
LGTM

File: shell/src/main/java/tachyon/shell/TfsShell.java [hunk 1]
```diff
@@ -93,7 +93,7 @@ private void loadCommands() {
         try {
           cmd = CommonUtils.createNewClassInstance(cls,
               new Class[] { TachyonConf.class, FileSystem.class },
-              new Object[] { mTachyonConf, mTfs });
+              new Object[] { mTachyonConf, mFileSystem});
```
Comment (apc999) [inline shell/src/main/java/tachyon/shell/TfsShell.java]: is the convention to have  a space before `{`?

Comment (calvinjia) [inline shell/src/main/java/tachyon/shell/TfsShell.java]: I would prefer not to but there is no checkstyle enforced.

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**